base/src/com/thoughtworks/go/util/SystemEnvironment.java

```diff
@@ -160,6 +160,11 @@
     public static GoSystemProperty<String> GO_LANDING_PAGE = new GoStringSystemProperty("go.landing.page", "/pipelines");
 
     public static GoSystemProperty<Boolean> FETCH_ARTIFACT_AUTO_SUGGEST = new GoBooleanSystemProperty("go.fetch-artifact.auto-suggest", true);
+    public static GoSystemProperty<String> GO_SSL_CONFIG_FILE_PATH = new GoStringSystemProperty("go.ssl.config.path", "/ssl.config");
+    public static GoSystemProperty<String> USER_CONFIGURED_SSL_CONFIG_FILE_PATH = new GoStringSystemProperty("user.ssl.config.path", "user.ssl.config");
+    public static GoSystemProperty<String> TRANSPORT_PROTOCOL_TO_BE_USED_BY_AGENT = new GoStringSystemProperty("agent.transport.protocol", "TLSv1.2");
```

Comment:
>> i thought SSLv3 was going to be the default

As @ketan mentioned, SSLv3 is not considered secure anymore. I am tempted to disable it by default (on server side), but decided against doing it as setups using java7 would stop working.  

>> do we know what is set of user agents (browsers) that are supported by the current transport protocol and ciphers? 

By default, since there are no restrictions, the server and client (agent/browser) could potentially decide on using an obsolete or a less secure cipher. Firefox and chrome usually complain about it. 
Anyway, I will try to put up the details you ask, it would be good to have that documented as I wont remember it after a few days :stuck_out_tongue: 

>> Also Java 7u25 does not support TLS v1.2, so we'd need to be careful about wording java 1.7 support.

Ack.